packs/github/actions/lib/base.py

```diff
@@ -16,11 +16,12 @@ def __init__(self, config):
         super(BaseGithubAction, self).__init__(config=config)
         token = self.config.get('token', None)
         token = token or None
-        self._client = Github(token)
+        base_url = self.config.get('base_url', None)
```

Comment:
That's fine for now.

I will go ahead, merge this into master and make the default change locally.

Thanks again :)